packages/tooling/fast-tooling/src/web-components/html-render-layer-inline-edit/html-render-layer-inline-edit.ts

```diff
@@ -17,6 +22,14 @@ export class HTMLRenderLayerInlineEdit extends HTMLRenderLayer {
     @attr({ mode: "boolean" })
     public autoselect: boolean;
 
+    /**
+     * Specifies when changes to the text should be committed.
+     * Only on "Enter" keypress or any time the textarea loses focus.
+     * Default: CommitMode.onBlur
+     */
+    @attr
+    public commitmode: CommitMode;
```

Comment:
Should we allow mutliple types of changes? `CommitMode[]`? Also wondering if this should be `commitMode` to follow your casing below.

Reply:
That was a bad suggestion, since this is an attr, we could do separate attrs `commitOnBlur` and `commitOnEnter` if we wanted to allow multiple commit modes.

Reply:
As per our offline discussion I have decided to change the commit modes to "onEnterOnly" and "onBlurOrEnter". The default behavior is now for the text changes to be committed any time the text area loses focus or if the Enter key is pressed. The "onEnterOnly" option will only commit when Enter is pressed and cancels the change on blur.